docs/articles/infrastructure/ci-cd/jenkins.rst

```diff
@@ -70,4 +70,43 @@ Normally we use the proposed plugins during installation and then add the follow
 - Copy Artifact Plugin
 - Build Monitor View
 
+Naming Jenkings jobs
+--------------------
+
+This part describes how to name Jenkins jobs in the Software Factory in a
+consistent manner. Intended reader is anyone that needs to add a Jenkins job.
+
+Syntax, using EBNF (Extended Backus-Naur Form @ Wikipedia) notation:
+    * job_name = component SEP job_type SEP build_interval
+    * build_interval = "nightly" | "hourly" | "weekly" | "on commit" | "manual"
```

Comment:
What about on release or if triggered by a different job which needs to run first but could be triggered on any kind of intervalls?